config/notifiers.go

```diff
@@ -92,6 +92,17 @@ var (
 		// TODO: Add a details field with all the alerts.
 	}
 
+	// DefaultVictorOpsConfig defines default values for VictorOps configurations.
+	DefaultVictorOpsConfig = VictorOpsConfig{
+		NotifierConfig: NotifierConfig{
+			VSendResolved: true,
+		},
+		MessageType:  `{{ if eq .Status "firing" }}CRITICAL{{ else }}RECOVERY{{ end }}`,
```

Comment:
Yes. Depends on the value, VictorOps will create or close an incident.  
It expects one of the following keywords in this field: INFO, WARNING, ACKNOWLEDGEMENT, CRITICAL, RECOVERY. Available here: [API Docs](http://victorops.force.com/knowledgebase/articles/Integration/Alert-Ingestion-API-Documentation/). 

Is there maybe a better way to resolve the exact status/type of an alert?